internal/cache/file/cache_handle_test.go

```diff
@@ -82,8 +83,10 @@ func (cht *cacheHandleTest) addTestFileInfoEntryInCache() {
 
 func (cht *cacheHandleTest) verifyContentRead(readStartOffset int64, expectedContent []byte) {
 	fileStat, err := os.Stat(cht.fileSpec.Path)
+	dirStat, err := os.Stat(filepath.Dir(cht.fileSpec.Path))
```

Comment:
Add AssertEq(nil, err) for "err" in fileStat, err also.

Reply:
done